package.json

```diff
@@ -31,34 +31,34 @@
   ],
```

Comment:
IMO this should come with a version bump to this module 
https://github.com/nodejs/node-core-utils/blob/f033641787f3595e4fedb184744ec6f7e52ca15c/package.json#L3

Reply:
What do you mean? This module's version will be bumped when the next release is done